concent-builder/test.yml

```diff
@@ -0,0 +1,37 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - consts.yml
+    - "{{ deployment_values }}/var.yml"
+    - "{{ deployment_values }}/var-{{ cluster }}.yml"
+  tasks:
+    - become:      yes
+      become_user: "{{ shared_user }}"
+      block:
+        - name: Configure kubectl to operate on the right cluster
+          command: gcloud container clusters get-credentials \
+              "{{ gke.cluster }}"                            \
+              --project "{{ gke.project }}"                  \
+              --zone    "{{ gke.zone }}"
+
+        - name:    Run custom network to allows communication between django and postgres containers
+          command: docker network create {{ image_prefix }}network
+
+        - name:    Run postgres container for django unit tests
+          command: docker run --rm --detach --net={{ image_prefix }}network --name {{ image_prefix }}postgresql-unittest {{ image_prefix }}postgresql:latest
+
+        - name:    Run concent-api container and django unit tests
+          command: docker run --rm                                                                                                                          \
+              --volume {{ build_dir }}/concent-deployment/kubernetes/build/run-unit-tests.sh:/usr/local/bin/run-unit-tests.sh                               \
+              --volume {{ build_dir }}/concent-deployment/kubernetes/build/docker_testing.py:/srv/http/concent_api/concent_api/settings/docker_testing.py   \
+              --net={{ image_prefix }}network concent-concent-api:latest /usr/local/bin/run-unit-tests.sh
+
+        - name:    Stop postgres container after tests were completed
+          command: docker stop {{ image_prefix }}postgresql-unittest
+
+        - name:    Delete custom network after tests were completed
+          command: docker network rm {{ image_prefix }}network
+      rescue:
```

Comment:
Instead of using `rescue` and repeating cleanup twice you could use `always` which is guaranteed to be always executed.

Reply:
Done.